examples/installation_on_windows/README.md

```diff
@@ -53,4 +53,4 @@ If it shows error message like 'error while loading shared libraries: libcudart.
 
 ### 3.2 Folder Access Issue
 
-If an error occurred while saving the model: No such file or directory: './models/vllm', it is probably because the `./models' folder created on the WSL connection are not accessible. Running `python3 ./examples/installation_on_windows/model_folder_checker.py` does a diagnosis on this access issue. (Make sure you are in the git cloned folder `ServerlessLLM`)
\ No newline at end of file
+If an error occurred while saving the model: No such file or directory: `./models/vllm`, it is probably because the `./models` folder created on the WSL connection are not accessible. Running `python3 ./examples/installation_on_windows/model_folder_checker.py` does a diagnosis on this access issue. (Make sure you are in the git cloned folder `ServerlessLLM`)
```

Comment:
Could you please clarify and reorganize the instructions? For running the script, it's preferable to use the following format:  
```bash
cd /path/to/xxx  
python xxx.py  
```  
Actually, the location of `.` for `./models/vllm` can be different from `./examples/installation_on_windows/model_folder_checker.py` if the user did not intially `cd` to the folder `ServerlessLLM`, which creates ambiguity.

Reply:
Changes have been made accordingly